src/VisualStudio/IntegrationTest/TestUtilities/InProcess/VisualStudioWorkspace_InProc.cs

```diff
@@ -81,5 +83,23 @@ public void CleanUpWaitingService()
 
                 GetWaitingService().EnableActiveTokenTracking(true);
             });
+
+        public void SetFeatureOption(string feature, string optionName, string language, string valueString)
```

Comment:
@rchande the framework should be resetting VS to its default options in between each scenario. So you shouldn't have to do anything "extra". However, if it isn't getting reset properly, then we need to fix that 😄